analysis/testing_params.R

```diff
@@ -0,0 +1,250 @@
+# Running Hector with parameter samples from Matilda
+library(hector)
+library(ggplot2)
+library(ncdf4)
+library(tidyverse)
+library(grid)
+library(gridExtra)
+
+# Set working directory
+setwd("C:/Users/done231/OneDrive - PNNL/Documents/GitHub/xploring-facts-data")
```

Comment:
Instead of hard-coding paths we should be using relative pathways, this let's the code be reproducible regardless of machines, I am happy to talk about best coding practices with you